### ISSUE TITLE
Add Wing to Serverless Land patterns

### ISSUE THREAD

**Issue by lordsilver** [winglang/wing#2207]
### Feature Spec

I think Wing should be added to the [Serverless Patterns Collection](serverlessland.com/patterns).

### Use Cases

Easily search for well-known serverless patterns written in Wing. Learning by examples is the best thing in my opinion so having some code samples you can rapidly filter on based on your needs would be nice.

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

**Comment by ShaiBer** [comment]
They currently only support SAM or CDK.
I guess we should talk to them to add support for Wing after the Beta (don't think they will add support for a language that says you can't really build anything with it :) )

**Comment by garysassano** [comment]
They support Pulumi and Terraform as well. And I think @singledigit will be happy to add Wing after the Beta.

**Comment by ShaiBer** [comment]
Cool!

**Comment by singledigit** [comment]
Yes, once Wing reaches beta I would love to add a Wing category. Plan on it!

**Comment by ShaiBer** [comment]
Keep

**Comment by revitalbarletz** [comment]
@shaiber01 - how do we want to proceed with this one?

**Comment by boyney123** [comment]
@singledigit would love to get this going ;)!
